pkg/network/placement_configuration.go

```diff
@@ -53,6 +53,7 @@ func GetDefaultPlacementConfiguration() cnao.PlacementConfiguration {
 		Workloads: &cnao.Placement{
 			NodeSelector: map[string]string{
 				"beta.kubernetes.io/arch": "amd64",
+				"kubernetes.io/os":        "linux",
```

Comment:
What if a user overrides the placement configuration.

Meni raised that it would be desirable to clearly fail NNCP if it cannot configure the policy due to Windows nodes matching it.